pretender.js

```diff
@@ -121,20 +189,6 @@ function verbify(verb){
   };
 }
 
-function throwIfURLDetected(url){
-  var HTTP_REGEXP = /^https?/;
-  var message;
-
-  if(HTTP_REGEXP.test(url)) {
-    var parser = window.document.createElement('a');
-    parser.href = url;
-
-    message = "Pretender will not respond to requests for URLs. It is not possible to accurately simluate the browser's CSP. "+
-              "Remove the " + parser.protocol +"//"+ parser.hostname +" from " + url + " and try again";
-    throw new Error(message)
-  }
-}
```

Comment:
@trek This seems to be directly in conflict with what I'm trying to do with this PR, so I just removed the check. Please let me know if you feel there's some way we can have our cake and eat it too.